packages/provider/src/api/captcha.ts

```diff
@@ -215,8 +129,13 @@ export function prosopoRouter(env: ProviderEnvironment): Router {
      */
     router.post(ApiPaths.GetPowCaptchaChallenge, async (req, res, next) => {
         try {
-            const { user, dapp } = GetPowCaptchaChallengeRequestBody.parse(req.body)
-
+            const { userAccount, dappAccount } = req.body
+            // Assert that the user and dapp accounts are strings
+            if (typeof userAccount !== 'string' || typeof dappAccount !== 'string') {
```

Comment:
Please use zod to parse and use ApiParams to build the zod parse.

Check the endpoint for SubmitCaptchaSolution to see how this is done

Reply:
Fixed